build.sh

```diff
@@ -4,6 +4,19 @@
 
 # Get directory this script is in
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+OS=$(uname -s)
+
+if [ "$OS" = "Darwin" ]; then
+    DIR_OS="MacOS"
+else
+    DIR_OS="Linux"
+fi
+
+if [[ "$*" == *"--ios"* ]]; then
+    DIR_OS="iOS"
+elif [[ "$*" == *"--android"* ]]; then
+    DIR_OS="Android"
+fi
 
 #requires python3.6 or higher
-python3 $DIR/tools/ci_build/build.py --build_dir $DIR/build/Linux "$@"
+python3 $DIR/tools/ci_build/build.py --build_dir $DIR/build/$DIR_OS "$@"
```

Comment:
>$DIR_OS [](start = 60, length = 7)

We may need to update some documents to reflect this change

Reply:
Went through all doc and none mentioned the output folder will be Linux.
And it should be ok since the change only happens macOS, android and iOS build, which should not impact the old scenario.